packages/tool-cache/package.json

```diff
@@ -36,7 +36,7 @@
     "url": "https://github.com/actions/toolkit/issues"
   },
   "dependencies": {
-    "@actions/core": "^1.2.0",
+    "@actions/core": "^1.2.3",
```

Comment:
required for isDebug: [Make extract functions quiet by default](https://github.com/actions/toolkit/pull/206)